### PR TITLE
AF-1798 : Standalone Decision Central package should provide better default configuration

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/AF-1798

### CODE REVIEW

**Comment by Rikkola** [comment]
@domhanak Hi, your marked as the tester for this.

File: business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  kie:
+    server: # Sample connection to KIE Server.
+      id: kieserver1
+      user: kieserver
+      pwd: kieserver1!
+      location: http://localhost:8230/kie-server/services/rest/server
+      controller:
+        user: kieserver
+        pwd: kieserver1!
+      controller: http://localhost:8080/business-central/rest/controller
```
Comment (Rikkola) [inline business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml]: @domhanak There is a small concern about if this works. Ideally the controller branch should not break, but if I have the url in the higher "controller:" I get an error on startup.

**Comment by domhanak** [comment]
Roses are red,
violets are blue,
ignorance is bliss,
Jenkins retest this

**Comment by domhanak** [comment]
Roses are red,
violets are blue,
Jenkins execute full downstream build!

**Comment by domhanak** [comment]
jenkins retest this

**Comment by domhanak** [comment]
jenkins retest this

**Comment by Rikkola** [comment]
jenkins retest this

**Comment by Rikkola** [comment]
@etirelli @mbiarnes This turned green. It should go in before the tag, but @domhanak has not had time to test yet. It works for me locally and @gitgabrio has also tried a local build. This build has no affect on the other builds, so even if it is bad it can not break anything.

File: business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      security:
+        management:
+          wildfly:
+            cli:
+              user: admin
+  kie:
+    server: # Sample connection to KIE Server.
+      id: kieserver1
+      user: kieserver
+      pwd: kieserver1!
```
Comment (domhanak) [inline business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml]: this does not match the password in `application-users.properties`  - there is `kieserver` as password in mentioned file

File: business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml (hunk 1 above)
Comment (domhanak) [reply business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml]: the `controller` tags here do not make any difference and the correct url would be `http://localhost:8080rest/controller`

Comment (domhanak) [reply business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml]: the `controller` tags here do not make any difference and the correct url would be `http://localhost:8080rest/controller`

File: business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml (hunk 2 above)
Comment (Rikkola) [reply business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml]: Thanks. I'll make a fix.

**Pull request merged**